src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -668,4 +685,31 @@ public static boolean isTest(@NonNull String shipperName) {
         }
         return true;
     }
+
+
+    // todo  arz fixme read from config file
```

Comment:
@MocanaAtBroad to  answer  your question: yes, I'm sure that hardcoding the origin values  is a bad idea  :)